lib/opencensus/trace/span_context.rb

```diff
@@ -78,6 +87,35 @@ def root
         root
       end
 
+      ##
+      # Returns true if this context equals or is an ancestor of the given
+      # context.
+      #
+      # @return [boolean]
+      #
+      def contains? context
+        until context.nil?
+          return true if context == self
+          context = context.parent
+        end
+        false
+      end
+
+      ##
+      # Returns all SpanBuilder objects created by this context or any
+      # descendant context. The order of the returned spans is undefined.
+      #
+      # @return [Array<SpanBuilder>]
+      #
+      def contained_span_builders
```

Comment:
Does this need to be part of the public facing surface area? We could leave it public and annotate it to hide from docs if necessary.

Reply:
Made them private for now. I guess they're not needed yet.